aim/web/ui/src/components/ChartPanel/ChartGrid/ChartGrid.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Grid, GridSize } from '@material-ui/core';
+
+import ErrorBoundary from 'components/ErrorBoundary/ErrorBoundary';
+import { chartTypesConfig } from 'components/ChartPanel/config';
+
+import chartGridPattern from 'config/chart-grid-pattern/chartGridPattern';
+
+import { IChartGridProps } from './ChartGrid.d';
+
+import './ChartGrid.scss';
+
+function ChartGrid({
+  data,
+  chartType,
+  chartRefs = [],
+  nameKey,
+  chartProps,
+  readOnly = false,
+  syncHoverState,
+}: IChartGridProps): React.FunctionComponentElement<React.ReactNode> {
+  return (
+    <ErrorBoundary>
+      {data.map((chartData: any, index: number) => {
+        const Component = chartTypesConfig[chartType];
+        const gridSize =
+          data.length > 9
```

Comment:
Create constants for gridSize

Reply:
Do you have any ideas about constant name ?